nexus/src/app/sagas/instance_common.rs

```diff
@@ -326,7 +329,10 @@ pub async fn instance_ip_get_instance_state(
                 ),
             )));
         }
-        (InstanceState::Vmm, Some(VmmState::Destroyed)) => {
+        (
+            InstanceState::Vmm,
+            Some(VmmState::Destroyed | VmmState::SagaUnwound),
```

Comment:
I think we should treat `VmmState::SagaUnwound` the same as `InstanceState::NoVmm` in this match. They describe the same condition: the instance has no active VMM and isn't even prospectively running anywhere; the difference is just in whether the updater task has garbage-collected the VMM ID left behind by the failed attempt to start.

Reply:
Yup, this makes sense to me. Will change it!